core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -38,9 +41,60 @@
 
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
+    private static volatile DistributionStatisticConfig distributionStatisticConfig =
+            DistributionStatisticConfig.builder()
+                                       .percentiles(PERCENTILES)
+                                       .build();
+
+    /**
+     * Sets {@link DistributionStatisticConfig} to {@link MoreMeters}.
+     * Any {@link Timer} and {@link DistributionSummary} which created by {@link MoreMeters}
```

Comment:
- Timer -> Timers
- DistributionSummary -> DistributionSummarys (`{@link DistributionSummary}s`)
- which created -> which are created
- by MoreMeters using ... and ... -> `by {@link #newTimer(...)} and {@link #newDistributionSummary(...)}`